packages/mobile/src/permissions/android-permissions.ts

```diff
@@ -0,0 +1,26 @@
+import { PermissionsAndroid } from 'react-native';
```

Comment:
I'd suggest using [react-native-permissions](https://github.com/zoontek/react-native-permissions) instead. We might need this in future

Also all this file belongs to `~helpers` folder, I don't think there is any value in creating a separate folder for permissions

Reply:
updated